subprojects/gradle/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/reservation/client/kubernetes/KubernetesReservationClient.kt

```diff
@@ -126,13 +124,13 @@ class KubernetesReservationClient(
                     )
                     serialsChannel.send(serial)
 
-                    logger.info("Pod ${pod.metadata.name} sent outside for further usage")
+                    logger.debug("Pod ${pod.metadata.name} sent outside for further usage")
                 } else {
-                    logger.info("Pod ${pod.metadata.name} can't load device. Disconnect and delete")
+                    logger.warn("Pod ${pod.metadata.name} can't load device. Disconnect and delete")
```

Comment:
As I understand it is a bad scenario when we couldn't boot device

Reply:
Hope to fix natively it by `startupProbe` in Kubernetes.
Unfortunately, it's available in 1.18+.